browser/js/app.js

```diff
@@ -8,6 +8,19 @@ app.config(function ($urlRouterProvider, $locationProvider) {
     $urlRouterProvider.otherwise('/');
 });
 
+// Create cookie controller
+
+app.controller('CookieCtrl', ['$scope','$cookieStore', function($scope, $cookieStore) {
```

Comment:
Yeah apparently, but I couldn't get it to work with `$cookies` and all the docs I could find were with `cookieStore`